cpp/ps0005/PS-5.cpp

```diff
@@ -4,42 +4,53 @@ using namespace std;
 
 int main()
 {
+    // prompting user for 2 operands and an operator 2/2
     float num1, num2;
     char op;
-    cout<< "please enter number1 \n";
-    cin>>num1;
-        cout<< "please enter the operator type + ,-,*,/ \n";
-    cin>>op;
-cout<< "please enter number2 \n";
-    cin>>num2;
-
-
+    cout << "please enter number1\n";
+    cin >> num1;
+    cout << "please enter the operator type +, -, *, /, %\n";
+    cin >> op;
+    cout << "please enter number2\n";
+    cin >> num2;

```

Comment:
- [x] declared variables to store the user input
- [x] chose suitable data types for the variables
- [x] prompted user to enter data
- [x] used `std::cin` in a right way to get user's input